platform/feature-api/api/pod-api/src/api.ts

```diff
@@ -131,22 +131,25 @@ export interface PolyOut extends Omit<FS, "readdir"> {
  */
 export interface PolyNav {
     /**
-     * A way for features to display a contents of a web page
+     * A way for features to display a contents of a web page of the given URL.
+     * @param {string} url - The URL to open.
      */
     openUrl(url: string): Promise<void>;
     /**
-     * Describe what actions are possible within the pod when a feature is loaded
+     * Describe what keyboard navigation actions are possible within the pod when a feature is loaded
+     * @param {string[]} actions - A list of actions that the user can take.
      */
     setActiveActions(actions: string[]): Promise<void>;
     /**
      * Set a title in of a Pod
+     * @param {string} title - The title to set.s
      */
     setTitle(title: string): Promise<void>;
     /**
-     * Ask the user to pick a file
-     * @param type the type of file the user is asked to select, as a valid MIME type string. If no type is passed, the user can chose any type of file.
+     * Ask the user to pick a file via a dialog and returns the selected file.
+     * @param {string} [type] - The type of file the user is asked to select, as a valid MIME type string. If no type is passed, the user can chose any type of file.
      * @throws if an unsupported MIME type was passed as the type argument.
-     * @return an ExternalFile Object or `null` if the user cancelled.
+     * @return The promise resolves with an ExternalFile Object or `null` if the user cancelled.
```

Comment:
Nit: Seems like so far we typically mean the promise's return values when we document `@return` - I don't see another case where we specifically say "The promise resolves with".